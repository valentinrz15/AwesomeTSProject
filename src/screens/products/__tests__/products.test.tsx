import React from 'react';
import renderer from 'react-test-renderer';
import { Products } from '../index';

describe('products test', () => {
  it('should render products snapshot', () => {
    const render = renderer.create(<Products />).toJSON();

    expect(render).toMatchSnapshot();
  });
});
