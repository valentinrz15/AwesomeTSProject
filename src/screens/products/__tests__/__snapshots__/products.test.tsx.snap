// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`products test should render products snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Products
  </Text>
</View>
`;
